packages/office-ui-fabric-react/src/components/Toggle/Toggle.deprecated.test.tsx

```diff
@@ -1,28 +1,18 @@
 import * as React from 'react';
 import { mount } from 'enzyme';
-import * as renderer from 'react-test-renderer';
-import * as sinon from 'sinon';
+import * as WarnUtil from '@uifabric/utilities/lib-commonjs/warn';
 
 import { Toggle } from './Toggle';
 
 describe('Toggle', () => {
-  it('renders a label', () => {
-    const component = mount(<Toggle label="Label" />);
-    expect(
-      component
-        .find('.ms-Toggle-label')
-        .first()
-        .text()
-    ).toEqual('Label');
-  });
-
-  it('renders toggle correctly', () => {
-    const component = renderer.create(<Toggle label="Label" />);
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+  beforeAll(() => {
+    // Prevent warn deprecations from failing test
+    jest.spyOn(WarnUtil, 'warnDeprecations').mockImplementation(() => {
+      /** no impl **/
+    });
```

Comment:
The afterAll should be here too

Reply:
ah, I was following a different example. will fix. :)